plugins/plugin-core-support/src/lib/cmds/history/history.ts

```diff
@@ -108,9 +108,10 @@ const showHistory = ({ argv, parsedOptions: options }) => {
   const filter = filterStr ? line => line.raw.indexOf(filterStr) >= 0 : () => true
 
   const startIdx = Math.max(0, historyModel.getCursor() - N - 1)
-  const endIdx = historyModel.getCursor() - 1
```

Comment:
We used to add command to history before command evaluation, but I changed this to add command&response to the history model after command evaluation. That's why the `endIdx` should just be `historyModel.getCursor()`